### PR TITLE
Approval flows

### PR DESCRIPTION
**Pull request by jamesachamp**
Request and recommendation approval flows have been created:

tbbo_approval_flow

Summary of changes:
- DB Schema / ORM:
  - Requests and Recommendations tables have a new boolean column named `is_approved`, which defaults to false
  - `get_all` method added to `core.py`
    - This returns all entries in a table that meet specific criteria
- Navigation:
  - Browse, request, and make recommendation links are not available if request path does not start with `/admin`
  - If the path does start with `/admin`, user is presented with the links seen in the above gif
  - Clicking on the header redirects to the homepage
- New Templates:
  - Separate templates for reviewing requests and recommendations have been created
  - Templates display information about all requests or recommendations that have not yet been approved
  - Title of each recommended book links to its Open Library works page
  - Each request or recommendation has both an approve and reject button
    - Approve button sets the model's `is_approved` field to true
    - Reject button deletes the model's entry from the database
- Styling:
  - Generic classes for buttons have been created, and new buttons utilize these classes
    - Previously existing buttons have not been changed
  - Added class for **bold** text

### CODE REVIEW

File: bestbook/views/__init__.py [hunk 1]
```diff
@@ -240,3 +269,37 @@ def get(self):
...
+
+    if resp.status_code == 200:
+        data = resp.json()
+        result['title'] = data['title']
+    else:
+        result['title'] = work_olid
+
+    result['link'] = "{}{}".format(base_url, work_olid)
+
+    return result
```
Comment (jamesachamp) [inline bestbook/views/__init__.py:305]: This function doesn't belong in this file.  Suggestions are welcome.

**Comment by jamesachamp** [comment]
If these changes are alright, the following DDL should be executed during deployment:

```
ALTER TABLE requests ADD COLUMN is_approved boolean;
ALTER TABLE requests ALTER COLUMN is_approved SET NOT NULL;

ALTER TABLE recommendations ADD COLUMN is_approved boolean;
ALTER TABLE recommendations ALTER COLUMN is_approved SET NOT NULL;
```

File: bestbook/views/__init__.py (hunk 1 above)
Comment (cclauss) [inline bestbook/views/__init__.py:305]:
```suggestion
    base_url = f"https://openlibrary.org/works/{work_olid}"
    response = requests.get(base_url + ".json")
    return {
        "title": response.json()["title"] if resp.status_code == 200 else work_olid
        "link": base_url
    }
```

**Review by cclauss — APPROVED** [review]
LGTM

File: bestbook/views/__init__.py (hunk 1 above)
Comment (jamesachamp) [reply bestbook/views/__init__.py]: Nice and elegant, thanks!  I've updated the code, but didn't use an f-string literal as this is currently being ran with Python 3.5.2 in the OJF server.

Comment (mekarpeles) [reply bestbook/views/__init__.py]: We may want to put this in the apis/books.py file attached to the Book class as a `classmethod`

**Pull request merged**